custom_conf.py

```diff
@@ -47,6 +47,9 @@
 ## when linking to the documentation from another website (see https://ogp.me/)
 # The URL where the documentation will be hosted (leave empty if you
 # don't know yet)
+# NOTE: If no ogp_* variable is defined (e.g. if you remove this section) the
+# sphinxext.opengraph extension will be disabled. Please consult with upstream
+# before you do this.
```

Comment:
What do you mean with "consult with upstream"? Do you mean ask a TA if it makes sense for that project?

Reply:
Note: Yes, my thought was that the recommended extensions are recommended for a reason. By asking any maintainer of this project
1. the downstream projects gets feedback if they are using the project as intended.
2. maintainers of this project maybe discover a use case that should be supported while consulting a downstream project.

Question: Should I remove these phrases?

Reply:
Note: I removed all these remarks with c860cca8def2837cf743eb8548039cf0ba5e5e0f

Reply:
Yes, I think it makes sense to remove them. Of course we always like feedback, but we shouldn't instruct people to get in touch for every small adjustment they want to make (which is supported by the framework).